subprojects/composite-builds/src/integTest/groovy/org/gradle/integtests/composite/CompositeBuildBuildPathAssignmentIntegrationTest.groovy

```diff
@@ -82,6 +88,15 @@ class CompositeBuildBuildPathAssignmentIntegrationTest extends AbstractComposite
             ':nested',
             ':nested:buildLogic'
         ]
+
+        outputContains("Build name=:")
```

Comment:
🤔 Would it make sense to grab all the `Build name=` outputs and then assert all the build identities against a collection? That way, you could make sure that you asserted all of them.

Reply:
Good idea! I pushed the changes